pennylane/transforms/defer_measurements.py

```diff
@@ -26,6 +26,76 @@
 # pylint: disable=too-many-branches, too-many-statements
```

Comment:
Do we still need these pylint silencings?

Reply:
Not anymore. Forgot to remove them 😄